apps/package.json

```diff
@@ -243,6 +244,7 @@
     "@xterm/xterm": "^5.5.0",
     "aws-sdk": "2.1231.0",
     "blockly": "11.1.1",
+    "blueimp-md5": "^2.19.0",
```

Comment:
We already have an MD5 library, `md5`.

Reply:
Ah, whoops, thanks, @cat5inthecradle. For some reason, I thought this was server-side only.